app/models/category.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 class Category < ApplicationRecord
+  paginates_per 10
```

Comment:
I think we can create one constant with value of 10 for the max pages and use it for category, lessons, user...

Reply:
It will be nice, to have possibility to choose the number of records per page. It can be a constant, for example [10, 50, 100, 500]. This value can be setted on the frontend, so user can decide, how many records he want to see per page. Also there should be a default value, 10 for example.